src/System.Xml.XPath.XDocument/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      net463-Windows_NT;
+      <!-- netfx-Windows_NT; clashes with NETStandard.Library -->
```

Comment:
Based on our other comments I assume you will re-enable any of these that we still plan to ship a package for.